src/components/table/index.tsx

```diff
@@ -59,9 +59,10 @@ export function TableColumn(props: TableColumnProps): ReactElement {
 
 export interface TableProps
   extends React.TableHTMLAttributes<HTMLTableElement> {
-  children:
+  children?:
     | ReactElement<TableColumnProps>
     | Array<ReactElement<TableColumnProps>>;
+  noDataComponent?: React.ComponentType | null;
```

Comment:
Just wondering why isn't this called `emptyStateComponent`? The term is widespread enough that it would make more sense to me to call it that.

Reply:
Totally agree with that 👌 . Will make the change